src/current/v23.2/node-shutdown.md

```diff
@@ -855,19 +828,15 @@ Cockroach Labs recommends that you:
 
 - Client applications that connect to CockroachDB must handle disconnections gracefully, because from the point of view of the client, cluster maintenance is always a possibility.
 
-  - A client application's connection pool should have a maximum lifetime that is shorter than the Kubernetes deployment's `terminationGracePeriodSeconds` setting. Cockroach Labs recommends aligning the value for [server.shutdown.connection_wait](#server-shutdown-connection_wait) with the connection pool's maximum lifetime.
-  - If a client application is connected to a cluster node that is draining, it will be disconnected, and must include the ability to try to connect again. Refer to [Connection retry loop](#connection-retry-loop).
+  A client application's connection pool should have a maximum lifetime that is shorter than the Kubernetes deployment's `terminationGracePeriodSeconds` setting. Cockroach Labs recommends aligning the value for [server.shutdown.connection_wait](#server-shutdown-connection_wait) with the connection pool's maximum lifetime.
 
 <a id="decommissioning-and-draining-on-cockroachdb-dedicated"></a>
 
 ## Decommissioning and draining on CockroachDB {{ site.data.products.dedicated }}
 
-Most of the guidance in this page is most relevant to manual deployments, although decommissioning and draining work the same way behind the scenes in a CockroachDB {{ site.data.products.dedicated }} cluster. CockroachDB {{ site.data.products.dedicated }} clusters have a termination grace period of 300 seconds (5 minutes). This setting is not configurable.
-
-Client applications that connect to CockroachDB must handle disconnections gracefully, because from the point of view of the client, cluster maintenance is always a possibility.
+Most of the guidance in this page is most relevant to manual deployments, although decommissioning and draining work the same way behind the scenes in a CockroachDB {{ site.data.products.dedicated }} cluster. CockroachDB {{ site.data.products.dedicated }} clusters have a termination grace period of 1800 seconds (30 minutes) and a `server.shutdown.connection_wait` value of 1800 seconds. These settings are not configurable.
```

Comment:
> clusters have a termination grace period of 1800 seconds (30 minutes) and a `server.shutdown.connection_wait` value of 1800 seconds.

we may get questions on this, since things won't work correctly if they are actually the same value.

earlier Carlo said:
> [We are setting the terminiation grace period to 45 minutes] but this is an internal value that accounts for drain_wait, 2*query_wait, and the lease transfer time. To customers we're communicating 30m.

maybe in these docs we can say:
> clusters have a `server.shutdown.connection_wait` value of 1800 seconds (30 minutes), and a termination grace period that is slightly longer than that.

also, technically "These settings are not configurable" is only true for the termination grace period. the other one is configurable, but we just wouldn't recommend changing it.

Reply:
Done